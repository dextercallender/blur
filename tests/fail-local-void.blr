void main() {
    void a;
    void b;
}
