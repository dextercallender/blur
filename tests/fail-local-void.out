Fatal error: exception Failure("illegal void variable a")
